### ISSUE TITLE
Memory graph flatline on FreeBSD

### ISSUE THREAD

**Issue by grahamperrin** [htop-dev/htop#1040]
I'm not sure when this issue began, but I no longer get any reading in the Mem graph: 

![image](https://user-images.githubusercontent.com/192271/174535204-5f5e2d1b-531f-4fe7-8ce0-abc6aed33ab4.png)

That is, htop from the FreeBSD repository. Not a custom build. 

```text
% pkg info -x htop
bashtop-0.9.25_1
htop-3.2.1
% pkg -vv | grep -e url -e enabled -e priority
    url             : "pkg+http://pkg0.pkt.freebsd.org/FreeBSD:14:amd64/latest",
    enabled         : yes,
    priority        : 2,
    url             : "https://alpha.pkgbase.live/current/FreeBSD:14:amd64/latest",
    enabled         : no,
    priority        : 0,
    url             : "file:///usr/local/poudriere/data/packages/main-default",
    enabled         : yes,
    priority        : 3
% uname -aKU
FreeBSD mowa219-gjp4-8570p-freebsd 14.0-CURRENT FreeBSD 14.0-CURRENT #13 main-n256147-1c665e95d44-dirty: Wed Jun 15 07:49:44 BST 2022     root@mowa219-gjp4-8570p-freebsd:/usr/obj/usr/src/amd64.amd64/sys/GENERIC-NODEBUG amd64 1400061 1400061
% 
```

**Comment by grahamperrin** [comment]
I wondered whether something in (bleeding edge) FreeBSD 14.0-CURRENT triggered the issue. Nope, it's reproducible on 13.0-RELEASE: 

![image](https://user-images.githubusercontent.com/192271/174706894-e6fa0bde-50bf-47db-9266-74baa29b69f3.png)

Above: custom build. 

Below: htop from the FreeBSD repository. 

![image](https://user-images.githubusercontent.com/192271/174707068-2a0e150c-e166-4135-a798-25a1da4c0bb8.png)

**Comment by grahamperrin** [comment]
Confirmed fixed in the port of 3.2.2. Thanks!
